### PR TITLE
Fixing the quiz titles to return correctly with specs

### PR DESCRIPTION
**Pull request by kamijean**

### CODE REVIEW

File: lib/senkyoshi/models/question_bank.rb [hunk 1]
```diff
@@ -28,7 +28,7 @@ def create_items(question_bank, resources)
       question_bank.questions = []
       questions.each do |item|
         question = item.canvas_conversion(question_bank, resources)
-        question.material = clean_up_material(question)
+        clean_up_material(question)
```
Comment (bfcoder) [inline lib/senkyoshi/models/question_bank.rb]: Consider this instead? Try to keep it pure?
```ruby
question.material = clean_up_material(question.material)
```

```ruby
TAGS = {
  "<p><span size=\"2\" style=\"font-size: small;\">.</span></p>" => "",
  "<p>.</p>" => "",
}.freeze
TAGS_RE = Regexp.union(TAGS.keys)

def clean_up_material(material)
  if material
    material.gsub!(TAGS_RE, TAGS)
    material.strip!
  end
  material
end
```

Comment (nick-benoit14) [reply lib/senkyoshi/models/question_bank.rb]: Won't `gsub!` and `strip!` both modify the original string though?

Comment (bfcoder) [reply lib/senkyoshi/models/question_bank.rb]: you have a good point.
```ruby
material = material.gsub(TAGS_RE, TAGS)
material = material.strip
```
would be better

**Review by bfcoder — APPROVED** [review]
Looks good!

**Pull request merged**